app/javascript/components/ExpressDonation/ExpressDonation.js

```diff
@@ -64,6 +77,7 @@ export class ExpressDonation extends Component<Props, State> {
         // form will have the user's submitted values
         ...this.props.fundraiser.form,
       },
+      allowDuplicate: this.state.optForRedonation,
```

Comment:
I think this method is the exception in terms of `camelCase`, because this is what gets sent to the backend (and the rails backend isn't automatically converting `camelCase` to `snake_case`. So we should actually use `allow_duplicate`.